AndroidTest/app/src/main/java/cloudant/com/androidtest/keyprovider/KeyDataTests.java

```diff
@@ -0,0 +1,135 @@
+package cloudant.com.androidtest.keyprovider;
+
+import android.test.AndroidTestCase;
+
+import com.cloudant.sync.datastore.encryption.KeyData;
+
+public class KeyDataTests extends AndroidTestCase {
```

Comment:
I suggest at least one happy-path test that makes sure the values passed into the constructor are returned correctly in each getter.